### PR TITLE
Fix docu link in auth-provider.js and filepath in README.md

### PR DESCRIPTION
**Pull request by gospodima**

### CODE REVIEW

**Comment by Fleker** [comment]
Thanks for catching this.

The change looks good to me @gospodima, although you'll need to sign the CLA before the PR can be approved and merged.

**Comment by gospodima** [comment]
@Fleker, I just now signed it.

File: README.md [hunk 1]
```diff
@@ -148,11 +148,11 @@ to create the project in the Actions Console, enter your Assistant settings.
 
     * If running locally:
 
-          node smart-home-provider-cloud.js isLocal
+          node cloud/smart-home-provider-cloud.js isLocal
```
Comment (Fleker) [inline README.md]: This should be `npm start`, to be consistent with what it stated in an earlier section.

**Comment by gospodima** [comment]
@Fleker, fixed.

**Pull request merged**